Formula/swift-protobuf.rb

```diff
@@ -0,0 +1,35 @@
+class SwiftProtobuf < Formula
+  desc "Plugin and runtime library for using protobuf with Swift"
+  homepage "https://github.com/apple/swift-protobuf"
+  url "https://github.com/apple/swift-protobuf/archive/0.9.904.tar.gz"
+  sha256 "d65970c3f17cb4a09e61102d7dce37733ad34fe723cf65795e856ec2cd126711"
+  revision 1
+  head "https://github.com/apple/swift-protobuf.git"
+
+  depends_on :xcode => ["8.3", :build]
+  depends_on "protobuf"
+
+  conflicts_with "protobuf-swift",
+    :because => "both install `protoc-gen-swift`"
+
+  def install
+    system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib"
+    bin.install ".build/release/protoc-gen-swift"
+  end
+
+  test do
+    testdata = <<-EOS.undent
+      syntax = "proto3";
+      enum Flavor {
+        CHOCOLATE = 0;
+        VANILLA = 1;
+      }
+      message IceCreamCone {
+        int32 scoops = 1;
+        Flavor flavor = 2;
+      }
+    EOS
+    (testpath/"test.proto").write(testdata)
+    system Formula["protobuf"].opt_bin/"protoc", "test.proto", "--swift_out=."
```

Comment:
Can we test that the output exists?

Reply:
Sure, I'll add that. Current test is exactly the same as for `protobuf-swift` formulae.